include/cynq/enums.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * See LICENSE for more information about licensing
+ *
+ * Copyright 2023
+ * Author: Luis G. Leon-Vega <luis.leon@ieee.org>
+ *         Diego Arturo Avila Torres <diego.avila@uned.cr>
+ *
+ */
+#pragma once
+
+/**
+ * @brief HardwareArchitecture
+ * Defines the architecture of the Create method, this is used by classes
+ * that implement the IHardware interface.
+ */
+enum class HardwareArchitecture {
+  UltraScale
+}
+
+/**
+ * @brief SyncType
+ * Declares the orientation of data synchronization in the Sync method,
+ * this is used by any class that implements the IMemory interface.
```

Comment:
For all the descriptions. Before `this`, put a point. `Sync method. This is used ...`